com.unity.render-pipelines.high-definition/Documentation~/Override-Fog.md

```diff
@@ -37,7 +37,7 @@ The **Fog** override of the active Volume controls the appearance of the global
 
 ## Properties
 
-![](Images/Override-VolumetricFog1.png)
+![](Images/fog-overrides.png)
```

Comment:
This screenshot should be in dark mode to conform with the image guidelines: https://unity-docs.gitbook.io/style-guide/format/images/screenshots

Reply:
As far as I can see, it is in dark mode